Providers/Modules/Plugins/Antimalware/plugin/collectmcafeeinfo.rb

```diff
@@ -9,34 +9,42 @@
 
 class McAfee
 
-	def self.detect()	
-		begin
-			if !File.file?('/opt/isec/ens/threatprevention/bin/isecav')
-				return false
-			end			
-			detectioncmd = `/opt/isec/ens/threatprevention/bin/isecav --version 2>&1`.lines.map(&:chomp)
+	attr_accessor :detectedPath , :mcafeeName , :mcafeeVersion 
+    def self.mcAfeeExist()
+        paths = ['/opt/isec/ens/threatprevention/bin/isecav', '/opt/McAfee/ens/tp/bin/mfetpcli']   
```

Comment:
Change order so new version is checked first.
Also consider renaming the function to "findmcafee"

Reply:
Sure I will update in next revision